packages/dotcom-server-handlebars/src/helpers/renderJsxComponent.ts

```diff
@@ -0,0 +1,36 @@
+import path from 'path'
+import React from 'react'
+import ReactDOMServer from 'react-dom/server'
+
+export default function renderJsxComponent({ hash }) {
```

Comment:
I think `renderReactComponent` would be more accurate as this is not generic.

Reply:
Fixed in [this commit](https://github.com/Financial-Times/dotcom-page-kit/pull/746/commits/e603923ae5ab1acde07b5a89b8b6c344adfc59e1).